MoltenVK/MoltenVK/Commands/MVKCmdAccelerationStructure.mm

```diff
@@ -0,0 +1,200 @@
+/*
+ * MVKCmdAccelerationStructure.mm
+ *
+ * Copyright (c) 2015-2023 The Brenwill Workshop Ltd. (http://www.brenwill.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "MVKCmdAccelerationStructure.h"
+#include "MVKCmdDebug.h"
+#include "MVKCommandBuffer.h"
+#include "MVKCommandPool.h"
+
+#include "MVKAccelerationStructure.h"
+
+#pragma mark -
+#pragma mark MVKCmdBuildAccelerationStructure
+
+VkResult MVKCmdBuildAccelerationStructure::setContent(MVKCommandBuffer*                                       cmdBuff,
+                                                      uint32_t                                                infoCount,
+                                                      const VkAccelerationStructureBuildGeometryInfoKHR*      pInfos,
+                                                      const VkAccelerationStructureBuildRangeInfoKHR* const*  ppBuildRangeInfos) {
+    VkAccelerationStructureBuildGeometryInfoKHR geoInfo = *pInfos;
+    
+    _infoCount = infoCount;
+    _geometryInfos = &geoInfo;
+    _buildRangeInfos = *ppBuildRangeInfos;
+    
+    return VK_SUCCESS;
+}
+
+void MVKCmdBuildAccelerationStructure::encode(MVKCommandEncoder* cmdEncoder) {
+    id<MTLAccelerationStructureCommandEncoder> accStructEncoder = cmdEncoder->getMTLAccelerationStructureEncoder(kMVKCommandUseNone);
```

Comment:
You should add an enumerator to `MVKCommandUse` for each of these new commands. This is used to set the debug label for the command encoder, which aids debugging.